crypto_analytics/collection/data_source/crypto_compare.py

```diff
@@ -1,28 +1,31 @@
 import pandas as pd
 import json
 import requests
+from typing import Dict, Union
 
-from crypto_analytics.collection.data_source import FinancialDataSource
-from crypto_analytics.types import Interval
+from crypto_analytics.collection.data_source import OHLCDataSource
+from crypto_analytics.types  import Interval
 
-class CryptoCompare(FinancialDataSource):
+class CryptoCompareOHLC(OHLCDataSource):
+    endpoints = {
+        Interval.MINUTE: 'data/histominute',
+        Interval.HOURLY: 'data/histohour',
+        Interval.DAILY: 'data/histoday',
+    }
```

Comment:
I thought it made sense to define this as a static/class variable because we don't need to define this for every instance of `CryptoCompareOHLC`. This seems to be a valid python pattern for class vars: https://www.geeksforgeeks.org/g-fact-34-class-or-static-variables-in-python/